service-locator/core/src/test/scala/com/lightbend/lagom/internal/client/ServiceNameMapperSpec.scala

```diff
@@ -8,70 +8,111 @@ import com.typesafe.config.ConfigFactory
 import org.scalatest.Matchers
 import org.scalatest.WordSpec
 
-class ServiceNameParserSpec extends WordSpec with Matchers {
+class ServiceNameMapperSpec extends WordSpec with Matchers {
 
   private val defaultConfig = ConfigFactory.defaultReference().getConfig("lagom.akka.discovery")
-  private val parser = new ServiceNameParser(defaultConfig)
+  private val parser = new ServiceNameMapper(defaultConfig)
   private def createParser(config: String) =
-    new ServiceNameParser(ConfigFactory.parseString(config).withFallback(defaultConfig))
+    new ServiceNameMapper(ConfigFactory.parseString(config).withFallback(defaultConfig))
 
   "The ServiceNameParser" should {
```

Comment:
Above this point there are 3 references to `parser` that should be renamed:

 * `private val parser`
 * `def createParser`
 * `"The ServiceNameParser"``

Some of the tests below also use the terminology `parser` instead of `mapper` in the test name.